ipatests/test_integration/test_installation.py

```diff
@@ -16,25 +16,24 @@
 from datetime import datetime, timedelta
 
 import pytest
-from cryptography.hazmat.primitives import hashes
 from cryptography import x509 as crypto_x509
+from cryptography.hazmat.primitives import hashes
 
 from ipalib import x509
 from ipalib.constants import DOMAIN_LEVEL_0
 from ipalib.constants import IPA_CA_RECORD
 from ipalib.sysrestore import SYSRESTORE_STATEFILE, SYSRESTORE_INDEXFILE
-from ipapython.dn import DN
```

Comment:
Is this reorder forced by something?

Reply:
Yes, this order is encouraged by [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) and convention, that imports should be sorted in alphabetical order.

Reply:
Yes, it should but this change was not needed for this file to pass checks, was it?
I do not see this as an issue just asking.

Reply:
not needed, it just a part of "holding code up to certain standards"

Reply:
I recommend to avoid refactoring unless it is required to make code more readable or to pass a linter. It reduces unnecessary noise.